services/web/server/src/simcore_service_webserver/projects/db.py

```diff
@@ -703,12 +742,6 @@ async def delete_project(self, user_id: int, project_uuid: str):
         )
 
         async with self.engine.acquire() as conn, conn.begin():
```

Comment:
one question here. I guess if the project does not exist now we get a sqlalchemy error instead of a one ours. is this ok?